bigbluebutton/bigbluebutton
# fix(external-video): unsynched player when autoplay is blocked

### What does this PR do?

If the user duplicates his/her session and the autoplay is blocked, when clicking to play the video it should be synchronized, but that is not happening. Instead, the video plays unsynchronized. This PR ensures the synchronization will be executed on play.


### Closes Issue(s)

n/a

### Motivation



### How to test

1. Create a meeting.
2. Join the meeting as presenter.
3. Join the meeting as viewer.
4. On presenter, share an external video.
5. On viewer, right-click on the browser tab and select **Duplicate**.
6. On the new tab that will be opened, the browser will block the autoplay of the video.
7. Click the play button.
8. See the video play unsynchronized.
